conductor-client/src/main/kotlin/com/openlattice/datastore/services/EntitySetManager.kt

```diff
@@ -108,4 +109,10 @@ interface EntitySetManager {
 
     fun setupOrganizationMetadataAndAuditEntitySets(entitySet: EntitySet)
 
+    fun getAuthorizedNeighborEntitySetsToEntityTypes(
```

Comment:
I'm struggling to understand this name. What does `ToEntityTypes` mean? Using `To` usually implies a map, like A maps to B, but this is not returning a map. I recommend renaming.

Reply:
Ooh good call, I originally had this returning a map but that was since updated. I'll change the name